manifests/v/VentisMedia/MediaMonkey/5/VentisMedia.MediaMonkey.installer.yaml

```diff
@@ -1,20 +1,77 @@
+# Created using YamlCreate.ps1
 # yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
 PackageIdentifier: VentisMedia.MediaMonkey
 PackageVersion: 5
 MinimumOSVersion: 10.0.0.0
 FileExtensions:
+- 3gp
+- 3gpp
+- aac
+- adts
+- aif
+- aifc
+- aiff
+- alac
+- ape
+- apl
+- asf
+- asx
+- avi
+- cda
+- cue
+- divx
+- f4v
+- fla
+- flac
+- flv
+- ifo
+- isma
+- m1v
+- m2v
+- m3u
+- m3u8
+- m4a
+- m4b
+- m4p
+- m4v
+- mac
+- mkv
+- mmip
+- mov
 - mp3
+- mp4
+- mpc
+- mpe
+- mpeg
+- mpg
+- mpp
+- mpv
+- ogg
+- ogm
+- ogv
+- pla
+- pls
+- wav
+- webm
+- wm
+- wma
+- wmv
+- xvid
 InstallModes:
-  - interactive
-  - silent
-  - silentWithProgress
+- interactive
+- silent
+- silentWithProgress
 Installers:
-  - Architecture: x86
-    InstallerType: inno
-    InstallerUrl: https://www.mediamonkey.com/sw/MediaMonkey_5.0.0.2338.exe
-    InstallerSha256: 30545D90F20E5F6F402025798171D3F1D8025910F81F9B456E8EA94E9D4264EA
-    Scope: machine
-    InstallerLocale: en-US
-    UpgradeBehavior: install
+- InstallerLocale: en-us
+  Architecture: x86
+  InstallerType: inno
+  Scope: user
+  InstallerUrl: https://www.mediamonkey.com/sw/MediaMonkey_5.0.1.2431.exe
+  InstallerSha256: 3580C5BEE97860FABD5CCDCBDB532DE2C2AB31689D5643291D46B1ADC3F1A26C
+  InstallerSwitches:
+    Silent: /verysilent
+    SilentWithProgress: /silent
```

Comment:
I believe these aren't needed? Switches are automatically added by winget-cli depending on `InstallerType`.

Reply:
Thanks, I wasn't aware that these switches are added by default. I removed them 7a494179c459864e1539a0220f2e961c5cef389a in the latest update and verified that everything still works as intended.